sormas-base/setup/server-setup.sh

```diff
@@ -331,6 +335,12 @@ ${ASADMIN} create-javamail-resource --mailhost localhost --mailuser user --froma
 ${ASADMIN} create-custom-resource --restype java.util.Properties --factoryclass org.glassfish.resources.custom.factory.PropertiesFactory --property "org.glassfish.resources.custom.factory.PropertiesFactory.fileName=\${com.sun.aas.instanceRoot}/sormas.properties" sormas/Properties
 
 cp sormas.properties "${DOMAIN_DIR}"
+SORMAS_PROPERTIES=${DOMAIN_DIR}/sormas.properties
+if [[ ${LINUX} = true ]]; then
+  export SORMAS_PROPERTIES
+else
+  setx SORMAS_PROPERTIES ${SORMAS_PROPERTIES}
```

Comment:
This seems to persistently add the environment variable to the user environment, which is probably not intended

Reply:
This was actually my intention. I probably misunderstood this part in our discussion. I thought I should set an environment variable with the sormas.properties path so that it is easily accessible in the other scripts.

Reply:
Under windows this will probably only affect developers.
Who usually are not really happy when their environment is meddled with.
And `export` only sets the variable for SUB-processes, so with Linux this will not be available after the script has returned.
For this to work it has to be called via [source](https://ss64.com/bash/source.html).

Reply:
I considered the option of preserving the variables in the shell, but I thought that the scripts can be run at different times in different shells. However, if this is sufficient I will do it like this.